src/OrchardCore.Modules/OrchardCore.Email/Services/SmtpService.cs

```diff
@@ -125,7 +122,33 @@ private MimeMessage FromMailMessage(MailMessage message)
             return mimeMessage;
         }
 
-        private async Task SendOnlineMessage(MimeMessage message, SmtpClient client)
+        private bool CertificateValidationCallback(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
+        {
+            bool valid = sslPolicyErrors == SslPolicyErrors.None;
+            if (!string.IsNullOrWhiteSpace(_options.CertificateThumbprint))
+            {
+                valid = certificate.GetCertHashString().Equals(_options.CertificateThumbprint);
+            }
+
+            if (!valid)
+            {
+                _logger.LogError($"SMTP Server's certificate {certificate.Subject} issued by {certificate.Issuer} " +
```

Comment:
Logging shouldn't be used with interpolated strings. It handles `{argument}` by itself. Please take a look at other usages in OC